vital/util/transform_image.h

```diff
@@ -90,9 +90,9 @@ void transform_image( image_of<T>& img, OP op )
   bool wBh = std::abs(img.w_step()) < std::abs(img.h_step()),
        dBh = std::abs(img.d_step()) < std::abs(img.h_step()),
        dBw = std::abs(img.d_step()) < std::abs(img.w_step());
-  size_t w_idx = ( ! wBh ) + dBw,
-         h_idx = wBh + dBh,
-         d_idx = ( ! dBw ) + ( ! dBh );
+  size_t w_idx = static_cast<size_t>( ! wBh ) + static_cast<size_t>(dBw),
```

Comment:
As we discussed, this change looks fine but we really should consider a more robust way of doing this.